cmd/sys-mgmt-executor/main.go

```diff
@@ -0,0 +1,212 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+// Global variables
+var err error
+
+var usageStr = `
+Usage: ./main service operation		Start app with requested {service} and {operation}
+       -h							Show this message
+`
+
+const (
+	START      = "start"
+	STOP       = "stop"
+	RESTART    = "restart"
+	AppOpenMsg = "This is the docker-compose-executor application!"
+)
+
+// TODO: Externalize these by putting in a properties file.
+const (
+	CoreCommandServiceKey          = "edgex-core-command"
+	CoreDataServiceKey             = "edgex-core-data"
+	CoreMetaDataServiceKey         = "edgex-core-metadata"
+	ExportClientServiceKey         = "edgex-export-client"
+	ExportDistroServiceKey         = "edgex-export-distro"
+	SupportLoggingServiceKey       = "edgex-support-logging"
+	SupportNotificationsServiceKey = "edgex-support-notifications"
+	SupportSchedulerServiceKey     = "edgex-support-scheduler"
+)
```

Comment:
These keys are already available within edgex-go
https://github.com/edgexfoundry/edgex-go/blob/998562a7ead719ed2741c5174aa2ede2132b50dd/internal/constants.go#L26

Given that the executor is in the mono-repo there's no reason to not utilize these.

Reply:
Also excellent feedback @tsconn23 - I'll incorporate it (and remaining feedback items) shortly 👍